migrations/2023-12-11-075542_create_pm_fingerprint_table/up.sql

```diff
@@ -0,0 +1,19 @@
+-- Your SQL goes here
+
+CREATE TYPE "BlocklistDataKind" AS ENUM (
+    'payment_method',
+    'card_bin',
+    'extended_card_bin'
+);
```

Comment:
do we need to have this as a database enum. can be a string.

Reply:
We won't be having much changes around this enum, so it being enum in DB should be fine.

cc: @vspecky